fava/application.py

```diff
@@ -214,10 +214,20 @@ def should_collapse_account(account_name):
 
         return account_name in collapse_accounts
 
+    def sidebar_queries():
```

Comment:
This function shouldn't exist and the whole thing should be done in the template IMHO.